app/gameengine.py

```diff
@@ -3249,6 +3293,13 @@ def archive_hand_continuation():
                         target_cards = target_player.center
                     case "collab":
                         target_cards = target_player.collab
+                    case "center_and_x_backstage":
+                        # backstage damage is applied first, before center
+                        center_effect = deepcopy(effect)
+                        center_effect["target"] = "center"
+                        center_effect["multiple_targets"] = None
+                        self.do_effect(effect_player, center_effect)
```

Comment:
Rather than calling do_effect, which will potentially get crazy with continuations, I think you should add this effect to the front of the list instead.
The easiest way to implement this would be to just queue up 2 separate damage events instead of processing this one half in another effect and half in this effect.
You could actually even do this in the definition?

There is some weirdness here that I still haven't addressed.
If this damage is supposed to be done simultaneously (which I think it is?) technically they die simultaneously and the owner resolves the death order I believe.
So far there isn't actually anything where this matters I think.

Reply:
I initially made it so that the effect can be constructed with just using what we already have: two ``deal_damage`` connected with ``and``; and I was expecting the result to be ``center`` first before ``backstage`` but ``backstage`` is prioritized whenever you have to choose which ``backstage`` to target.

Which prompted me to create that new ``target`` value which makes it so ``center`` will be prioritized whenever there is no ``choice`` to be presented but along the way I decided to just do ``backstage`` first so it will be consistent throughout.

Reply:
So, stupid of me overcomplicating stuff. No need to use ``and``. Just make them separate effects one after the other. I'll take those things out.

Reply:
yes, i think separate effects works fine for now